scripts/build.py

```diff
@@ -130,8 +154,8 @@ def _build_files():
 
 def get_cache_slug():
     """Returns the cache slug read from file."""
-    with open('cache_slug.yaml', 'r') as f:
-        content = f.read()
+    with open('cache_slug.yaml', 'r') as _:
+        content = _.read()
```

Comment:
``_`` is generally used when we aren't going to use a variable in a  loop. Consider naming it something meaningful, instead:

``with open('cache_slug.yaml', 'r') as cache_slug_file:``